types/types.go

```diff
@@ -103,7 +103,8 @@ const (
 	// DefaultStaleReplicaTimeout in minutes. 48h by default
 	DefaultStaleReplicaTimeout = "2880"
 
-	EngineImageChecksumNameLength = 8
+	EngineImageChecksumNameLength          = 8
+	InstanceManagerImageChecksumNameLength = 44
```

Comment:
Why we keep 44 bytes of checksum? It's not necessary. 8 bytes like EngineImage checksum should be good enough. Both are the unique checksum for the image check, so we shouldn't treat them differently.

Reply:
I wanted to keep the instance-manager label length to be the same as before which is 48. Therefore, I set InstanceManagerImageChecksumNameLength = 44. Yes, it is unnecessary though. Will fix it.